ngi_pipeline/database/classes.py

```diff
@@ -117,6 +116,9 @@ def sample_get(self, projectid, sampleid):
     def sample_get_libpreps(self, projectid, sampleid):
         return self.get(self.construct_charon_url('libpreps', projectid, sampleid)).json()
 
+    def sample_get_projects(self, sampleid):
+        return self.get(self.construct_charon_url('projectidsfromsampleid', sampleid)).json()
```

Comment:
it's deployed and returns a list 100% of the time

On Fri, May 1, 2015 at 2:08 AM, Mario Giovacchini <notifications@github.com>
wrote:

> In ngi_pipeline/database/classes.py
> <https://github.com/NationalGenomicsInfrastructure/ngi_pipeline/pull/282#discussion_r29482185>
> :
>
> > @@ -117,6 +116,9 @@ def sample_get(self, projectid, sampleid):
> >      def sample_get_libpreps(self, projectid, sampleid):
> >          return self.get(self.construct_charon_url('libpreps', projectid, sampleid)).json()
> >
> > +    def sample_get_projects(self, sampleid):
> > +        return self.get(self.construct_charon_url('projectidsfromsampleid', sampleid)).json()
>
> @Galithil <https://github.com/Galithil> I think this is correct as well
> but let me know if I've got it wrong.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NationalGenomicsInfrastructure/ngi_pipeline/pull/282/files#r29482185>
> .
>